python/create_mask.py

```diff
@@ -0,0 +1,138 @@
+"""
+
+; Project     : Coronal Hole Identification via Multi-thermal Emission Reconition Algorithm (CHIMERA)
+;
+; Name        : create_mask
+;
+; Purpose     : Create transparent CH masks from previously stored data
+;
+; Syntax      : create_mask.py path 
```

Comment:
If you want to run it as such, then you need the first line of the file with a `#!/usr/bin/env python` and provide executable permissions to this file.